### PR TITLE
CM spec updates for ReSpec  2.1.6

### PR DESCRIPTION
**Pull request by berezovskyi**

### CODE REVIEW

**Comment by berezovskyi** [comment]
image

**Comment by berezovskyi** [comment]
image

File: specs/cm/change-mgt-vocab.html [hunk 1]
```diff
@@ -1,154 +1,127 @@
...
-
-    editors:  [
+  <meta charset='utf-8'/>
+  <title>OSLC Change Management Version 3.0. Part 2: Vocabulary</title>
+
+  <script src='https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.6/builds/respec-oasis-common.min.js' async class='remove'></script>
+  <!-- <script src='http://127.0.0.1:9000/respec-oasis-common-2.1.6.js' async class='remove'></script> -->
+
+  <script class='remove'>
+    var filePath = "/change-mgt-vocab.html";
```
Comment (jamsden) [inline specs/cm/change-mgt-vocab.html]: relative URL shouldn't start with /

Comment (berezovskyi) [reply specs/cm/change-mgt-vocab.html]: this is a JavaScript variable, used only internally. I also expect @ndjc to be not quite thrilled with my workaround :)

**Comment by berezovskyi** [comment]
@jamsden we will make a separate PR promoting it to the PSD once we got all the formatting right. You will get a chance to practice our agreed approach no. 2 :)

**Comment by berezovskyi** [comment]
> This version should be docs.oasis-open.org/oslc-op/cm/v3.0/ps01/change-mgt-spec.html, it is the URL to the OASIS published document you are looking at

to be clear, you are looking at a WD and that's why the URL is different.

**Comment by berezovskyi** [comment]
Jim, the properly formatted draft is live: https://oslc-op.github.io/oslc-specs/specs/cm/change-mgt-spec.html

You can now create a new PR promoting it to PSD. I will review it.

**Comment by jamsden** [comment]
I will start going through the issues today.

**Comment by jamsden** [comment]
Shouldn't it be docs.oasis-open-projects.org?

**Comment by berezovskyi** [comment]
> Shouldn't it be docs.oasis-open-projects.org?

WDs are published on https://oslc-op.github.io/oslc-specs, everything else under docs.oasis-open-projects.org.

> I will start going through the issues today.

You only have to change https://github.com/oslc-op/oslc-specs/pull/95/files#diff-5f252ec413838f13de10abe7c2958a33R15 to say `"PSD"` in a separate PR (which we will merge and tag `cm-v3.0-psd03x` because you previously pushed the tag `cm-v3.0-psd03` to this repo) unless there are other issues in the spec (though any material changes at this stage would require us to start from WD04 and PSD04 again) and then produce the ZIP archive for OASIS.

**Comment by berezovskyi** [comment]
> Shouldn't it be docs.oasis-open-projects.org?

Also, as you may see, the WD does not have a link "Current Stage" at all. The current stage link for a WD is the one you see in the address field of your browser.

**Pull request merged**